src/expression-helpers.js

```diff
@@ -37,6 +41,7 @@ function getObservableValue_fromDynamicKey_fromObservable(key, root, helperOptio
 	}, function setDynamicKey(newVal){
 		stacheKey.write(canReflect.getValue(root), getKeys(), newVal);
 	});
+	// This prevents lazy evalutaion
```

Comment:
I'm not sure what this means exactly. Removing this `temporarilyBind` doesn't seem to break anything.